app/Importer/UserImporter.php

```diff
@@ -38,7 +38,7 @@ public function createUserIfNotExists(array $row)
         $this->item['email'] = $this->findCsvMatch($row, 'email');
         $this->item['phone'] = $this->findCsvMatch($row, 'phone_number');
         $this->item['jobtitle'] = $this->findCsvMatch($row, 'jobtitle');
-        $this->item['activated'] = ($this->findCsvMatch($row, 'activated')) ? 1 : 0;
+        $this->item['activated'] = ($this->fetchHumanBoolean($this->findCsvMatch($row, 'activated')) == '1') ? '1' : 0;
```

Comment:
I think the `fetchHumanBoolean()` method should return 1 or 0, so we maybe don't need the ternary here?

Reply:
Yes... totally, I think the same the first time I see it. But just now erase it and test it. It still (of course) works.